helm-app/templates/users-service-db-service.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.service.name }}
+  namespace: {{ .Values.namespace }}
+spec:
+  selector:
+    app: {{ .Values.service.selector.app }}
+  ports:
+  - protocol: TCP
+    port: {{ .Values.service.ports.port }}
+    targetPort: {{ .Values.service.ports.targetPort }}
+  type: {{ .Values.service.type }}

```

Comment:
If I want to create a `NodePort` or `LoadBalancer` type service, will this let me?